src/Voron/Data/Tables/TableSchema.cs

```diff
@@ -328,31 +317,169 @@ public void Validate(FixedSizeSchemaIndexDef actual)
             }
         }
 
-        public TableSchema CompressValues(FixedSizeSchemaIndexDef etagSource, bool compress)
+        public class DynamicBTreeIndexDef : AbstractBTreeIndexDef
```

Comment:
As proposed above, maybe use `DynamicKeyIndexDef` name for that.

Reply:
done